uninstall.sh

```diff
@@ -1,7 +1,19 @@
-#!/bin/sh
+#!/bin/bash
 # uninstall by (github: bernardofire)
 # Remove line from bashrc and zshrc if present.
-for rc in bashrc zshrc; do
-  sed -i '/scm_breeze/d' "$HOME/.$rc"
-  printf "Removed SCM Breeze from %s\n" "$HOME/.$rc"
-done
+
+if [[ $OSTYPE == "Darwin" ]]; then 
```

Comment:
Needs to be `== "darwin"*`, on my shell `echo $OSTYPE` reports `darwin16`. See http://stackoverflow.com/questions/394230/detect-the-os-from-a-bash-script